### ISSUE TITLE
input type="range" does not slide on amp-lightbox

### ISSUE THREAD

**Issue by Alvillarroel** [ampproject/amphtml#19961]
I think it's a bug, the point is like post title said,
I have a page on amp and I want to show a form in amp-lightbox element with inputs to filter data, two of this inputs are type="range" to filter Years and Price.
but when I want to filter by these type of inputs does not slide. (this only happens on mobile version)

I added the same inputs type outside amp-lightbox element in same page and works as expected (Yes on mobile). 

the page to test is -> https://difor.mentalidadweb.cl/amp.html

I appreciate your helps.

**Comment by aghassemi** [comment]
@Alvillarroel It's a bug, you can workaround it for now by adding `scrollable` to lightbox.

**Comment by Alvillarroel** [comment]
@aghassemi thanks it works! 😃

**Comment by nainar** [comment]
Original URL doesn't work. But the bug should still repro

**Comment by alanorozco** [comment]
This works for me:

```html
  <button on="tap:lightbox">
    Open
  </button>
  <amp-lightbox id="lightbox" layout="nodisplay">
    <div>
      <button on="tap:lightbox.close">
        Close
      </button>
    </div>
    <input type="range">
  </amp-lightbox>
```

I'll close, but can reopen if provided with a repro case.
